features/step_definitions/web_steps.rb

```diff
@@ -31,7 +31,7 @@ def with_scope(locator)
 
 When /^(?:|I )follow( invisible)? "([^"]*)"(?: within "([^"]*)")?$/ do |invisible, link, selector|
   with_scope(selector) do
-    click_link(link, visible: !invisible)
+    click_link(link, exact: true, visible: !invisible)
```

Comment:
Oh yes this will make the page load a bit to find the right element before clicking
On slow ajax request the click_link is fatal, it catches "ApiDocs" instead of "Docs" because it is a regex match ...